server/cp/mgmt_api_signals.py

```diff
@@ -1,7 +1,7 @@
 from newsroom.signals import company_create
 
 
-def on_company_create(sender, company, **kwargs):
+async def on_company_create(sender, company, **kwargs):
```

Comment:
These, and the server/cp/signals, have a different signature with the new Async signals. They don't have `sender` anymore (was never used anyway). Otherwise these signals will raise an exception when called